graphql/user/fields.ts

```diff
@@ -76,8 +76,9 @@ export class UserFieldsResolver {
 
     @FieldResolver((returns) => [ConcreteNotification])
     @Authorized(Role.OWNER, Role.ADMIN)
+    @LimitedQuery()
```

Comment:
For this you also need `@Arg('take')` (and `@Arg('skip')`) (c.f. https://github.com/corona-school/backend/blob/master/graphql/subcourse/fields.ts#L65 for an analogous case)

Reply:
fixed.  
schoolForTeacherEmail in school/fields.ts was not using this args, so I didn't add them either first - I think the query won't make use of them

Reply:
@jeffersondarcy oh no, actually this is a bug and the usage of LimitedQuery there is wrong. Please pass on take and skip to Prisma  `findMany({ take, skip, where: { ... }, orderBy: { sentAt: 'desc' } })`. I just have a bad feeling exposing unlimited queries to users.

Reply:
fixed